source/fundamentals/connection/connect.txt

```diff
@@ -77,12 +77,34 @@ In the example, we set two connection options: ``maxPoolSize=20`` and
 
 .. _connect-atlas-java-driver:
 
-The following code shows how you can use the sample connection URI in a client
-to connect to MongoDB.
+Atlas Connection Example
+------------------------
+
+To connect to a MongoDB deployment on Atlas, create a client. You can
+create a client that uses your connection string and other 
+client options by passing a ``MongoClientSettings`` object to the
+``MongoClients.create()`` method.
+
+To instantiate a ``MongoClientSettings`` object, call the
+``MongoClientSettings.builder()`` method. You can chain methods to your
+``MongoClientSettings`` instance to specify your connection string and other
+client options.
+
+Chain the ``applyConnectionString()`` method to your ``MongoClientSettings``
+instance to specify your connection URI.
```

Comment:
Issue:
Sorry that I didn't identify this earlier. This should not be a MongoClientSettings instance, but rather a[ MongoClientSettings.Builder](https://mongodb.github.io/mongo-java-driver/4.9/apidocs/mongodb-driver-core/com/mongodb/MongoClientSettings.Builder.html) instance.

Suggestion:

Here's a simpler version that contains less info. I think it should still be clear enough since it's demonstrated in the code snippet.

"To instantiate a MongoClientSettings object, use the builder method to specify your connection string and any other client options, and then call the build() method. Chain the applyConnectionString() method to the builder to specify your connection URI."